perllib/FixMyStreet/DB/Result/Comment.pm

```diff
@@ -229,14 +229,18 @@ sub meta_line {
 
     if ($self->anonymous or !$self->name) {
         $meta = sprintf( _( 'Posted anonymously at %s' ), Utils::prettify_dt( $self->confirmed ) )
-    } elsif ($self->user->from_body) {
+    } elsif ($self->user->from_body || ( $self->name eq _('admin') && $self->user->is_superuser)) {
```

Comment:
Hmm, comparing the name against a string feels a bit wrong.
Perhaps we should record (in the update's extra) if someone is a staff user or superuser at the time they make the update - this would also mean if they stop being a superuser/staff user, that information is not lost. At present I would suggest they got a different account rather than switch the status of the user, iyswim. But that's probably quite a bigger change.
So perhaps this will do for now, what do you think?

Reply:
Yeah, I was a bit unsure about this too.